zppy/e3sm_diags.py

```diff
@@ -64,6 +64,29 @@ def e3sm_diags(config, scriptDir):  # noqa: C901
                     c["ref_year1"],
                     c["ref_year2"],
                 )
+                reference_data_path = (
```

Comment:
Now, these parameters are defined with `reference_data_path`. They can still be specified explicitly though.